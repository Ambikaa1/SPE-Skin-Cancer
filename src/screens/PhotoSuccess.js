import React from 'react';
import {StyleSheet, View, Text, TouchableOpacity, ScrollView} from 'react-native';

const PhotoSuccess = ({navigation}) => {
    return (
        <ScrollView>
            <Text style={styles.headingText}> Your mole photos are saved! </Text>
            <Text style={styles.bodyText}> Press finish below to go back to the body diagram. </Text>
            <TouchableOpacity style={styles.doneBox} onPress={() => navigation.navigate("Swiping")}>
                <Text style = {styles.doneText}>Finish</Text>
            </TouchableOpacity>
            <Text style={styles.bodyText}>{'\n'}Alternatively, you can now complete an optional questionnaire by
                pressing take questionnaire below.
                {'\n\n'}Your answers will be attached to any email you send through the app.
            It is made for people who have, or have had, a diagnosis of skin cancer.{'\n\n'}This questionnaire is
            strictly optional and you can change your answers by answering it again.</Text>
            <TouchableOpacity style = {styles.doneBox} onPress={() => navigation.navigate("Survay")}>
                <Text style = {styles.doneText}>
                    Take questionnaire
                </Text>

            </TouchableOpacity>
        </ScrollView>
    )
};

const styles = StyleSheet.create({
    headingText: {
        color: "black",
        fontSize: 25,
        paddingHorizontal:5,
        paddingVertical: 10,
        alignSelf: 'center',
    },
    bodyText:{
        color: "black",
        fontSize: 20,
        paddingHorizontal:5,
        paddingVertical:5,
    },
    doneBox: {
        backgroundColor: "#71A1D1",
        alignSelf: "center",
        borderRadius: 10,
        marginTop: 20,
        width: "97.5%",
    },
    doneText: {
        fontSize: 30,
        fontWeight: "bold",
        color: "white",
        marginVertical: 10,
        alignSelf:'center',
    }
});

export default PhotoSuccess;
