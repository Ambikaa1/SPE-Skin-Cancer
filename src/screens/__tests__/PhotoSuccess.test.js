import React from 'react';
import renderer from 'react-test-renderer';

import PhotoSuccess from "../PhotoSuccess";

describe('<PhotoSuccess />', () => {
    it('renders correctly', () => {
        const tree = renderer.create(<PhotoSuccess />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
