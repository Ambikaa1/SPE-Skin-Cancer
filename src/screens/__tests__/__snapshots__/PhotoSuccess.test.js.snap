// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PhotoSuccess /> renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
>
  <Text
    style={
      Object {
        "fontSize": 30,
        "fontWeight": "bold",
        "marginHorizontal": 10,
        "marginTop": 10,
      }
    }
  >
    Your mole photos have been saved!
  </Text>
  <Text
    style={
      Object {
        "fontSize": 17,
        "marginHorizontal": 10,
        "marginTop": 10,
      }
    }
  >
    Press the button below if you would like to schedule a notification to remind you to take a new picture of this mole in 30 days.
  </Text>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#71A1D1",
        "borderRadius": 10,
        "bottom": 10,
        "marginHorizontal": 10,
        "marginTop": 25,
        "opacity": 1,
      }
    }
  >
    <Text
      style={
        Object {
          "alignSelf": "center",
          "color": "white",
          "fontSize": 17,
          "fontWeight": "bold",
          "marginVertical": 15,
        }
      }
    >
      Press to schedule a notification
    </Text>
  </View>
  <Text
    style={
      Object {
        "fontSize": 17,
        "fontWeight": "bold",
        "marginHorizontal": 10,
        "marginTop": 10,
      }
    }
  >
    Would you now like to complete the SCQOLIT survey? 
  </Text>
  <Text
    style={
      Object {
        "fontSize": 17,
        "marginHorizontal": 10,
        "marginTop": 10,
      }
    }
  >
    The SCQOLIT is an optional questionnaire for people who have been diagnosed with skin cancer. 
  </Text>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignSelf": "center",
        "backgroundColor": "#71A1D1",
        "borderRadius": 10,
        "marginHorizontal": 10,
        "marginTop": 10,
        "opacity": 1,
        "width": "95%",
      }
    }
  >
    <Text
      style={
        Object {
          "alignSelf": "center",
          "color": "white",
          "fontSize": 17,
          "fontWeight": "bold",
          "marginVertical": 15,
        }
      }
    >
      Yes, take me to the SCQOLIT survey
    </Text>
  </View>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignSelf": "center",
        "backgroundColor": "#71A1D1",
        "borderRadius": 10,
        "marginHorizontal": 10,
        "marginTop": 10,
        "opacity": 1,
        "width": "95%",
      }
    }
  >
    <Text
      style={
        Object {
          "alignSelf": "center",
          "color": "white",
          "fontSize": 17,
          "fontWeight": "bold",
          "marginVertical": 15,
        }
      }
    >
      No, take me back to the body outline
    </Text>
  </View>
</View>
`;
